app/views/docs/functions.phtml

```diff
@@ -503,14 +503,64 @@ static RuntimeResponse &main(const RuntimeRequest &req, RuntimeResponse &res) {
 
 <p><a href="/docs/server/functions#functionsCreateDeployment" rel="noopener"><i class="icon-angle-circled-right margin-start-negative-tiny margin-end-tiny"></i> You can also create new code deployments using the Appwrite server API</a></p>
 
+<h3>Manual Deployment</h3>
+<p>You can also upload your functions to be deployed using the Appwrite console. The example below shows a typical Node.JS function, but the same idea applies to any other language.</p>
+
+<div class="ide margin-bottom">
+    <pre class="line-numbers"><code class="prism" data-prism>.
+├── package.json
+└── src
+    └── main.js
+</code></pre>
+</div>
+
+<p>First, package your code files into the <code>.tar.gz</code> format with the tar command:</p>
+
+<ul class="phases clear" data-ui-phases>
+    <li>
+        <h3>Unix</h3>
+
+        <div class="ide margin-bottom" data-lang="bash" data-lang-label="Bash">
+            <pre class="line-numbers"><code class="prism language-bash" data-prism>tar -zcvf code.tar.gz src package.json</code></pre>
```

Comment:
Better command: `tar -czf code.tar.gz --exclude code.tar.gz .`
(gzip everything in current directory EXCEPT gzip itself).

Reply:
😬 They're likely to catch stuff like .envs, node_modules, .vscode. all kinds of garbage like that.

Also they might run it in the wrong folder.

Reply:
Valid point. Feel free to revert to your previous solution of specifying files. We could ignore node_modules, but people could use same command for Python. Specifying feels fine. They shouldnt have too complex of a function anway. if they do, Appwrite CLI is a way to do it.

Reply:
Resolving for now. This should be fine. If we see people wondering why their manual deployments don't work/take too long/have packages track, we fix after.